### PR TITLE
community/wine_gecko: modernize

### PR DESCRIPTION
**Pull request by J0WI**
Not sure if this should be moved to unmaintained?

### CODE REVIEW

**Comment by Ikke** [comment]
@J0WI What is the reason this should be moved to unmaintained?

**Comment by J0WI** [comment]
There was no progress over the last ~3 years and I think it's very insecure if the browser engine has not been updated since then.
https://sourceforge.net/p/wine/wine-gecko/ci/master/tree/

**Comment by Ikke** [comment]
So it's more because upstream is unmaintained, not the package itself?

**Comment by J0WI** [comment]
> So it's more because upstream is unmaintained, not the package itself?

Yes

File: community/wine_gecko/APKBUILD [hunk 1]
```diff
@@ -3,30 +3,22 @@
...
 pkgver=2.47
-pkgrel=0
+pkgrel=1
 pkgdesc="Wine's built-in replacement for Microsoft's Internet Explorer"
-url="http://wiki.winehq.org/Gecko"
+url="https://wiki.winehq.org/Gecko"
 arch="x86 x86_64"
 license="MPL"
-options="!archcheck"
+options="!archcheck !check"
```
Comment (andypost) [inline community/wine_gecko/APKBUILD]: makes sense to mention why no tests possible

Comment (J0WI) [reply community/wine_gecko/APKBUILD]: Done

**Comment by andypost** [comment]
Commited as 5c9217e4

**Pull request closed without merging**